File: src/main/java/com/example/onemoretime/basic/PrimeNumber.java
package com.example.onemoretime.basic;

public class PrimeNumber {
    public static void main(String[] args) {
        int n = 14;
        // System.out.println(primeNumber(n));
       // primeNumber(n);
        allPrimeNumbersTillANumber(n);
    }

    static void primeNumber(int n) {
        boolean flag = false;
        for (int i = 2; i < n; i++) {
            if (n % i == 0) {

                flag = true;
                //return false;
                break;
            }
            //  System.out.println(i);
        }
        if (!flag) {
            System.out.println(n + "is prime");
        } else {
            System.out.println(n + "not prime");
        }
    }

    static void allPrimeNumbersTillANumber(int n) {
        for (int i = 2; i <= n; i++) {
            boolean isprime = true;
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    isprime = false;
                    break;
                }
            }
            if (isprime) {
                System.out.println(i + " is a prime number");
            } else {
                  System.out.println(i + " is not prime number");
            }
        }
    }
}

File: src/main/java/com/example/onemoretimekotlin/basic/PrimeNumber.kt
package com.example.onemoretimekotlin.basic

fun main(args: Array<String>) {
    //isPrime(8);
    allPrimeNumbersTillANumber(9)
}

fun isPrime(n: Int) {
    var flag = true;
    for (i in 2 until n) {
        if (n % i == 0) {
            flag = false
            break
        }

    }
    if (flag && n > 1) {
        println("$n  is prime  number")
    } else {
        println("$n  is prime not number")

    }

}

fun allPrimeNumbersTillANumber(n: Int) {
    for (i in 2..n) {
        var flag = true
        for (j in 2 until i) {
            if (i % j == 0) {
                flag = false
                break
            }
        }
        if (flag) {
            println("$i  is prime  number")

        } else {
            println("$i  is not  prime  number")

        }
    }
}